modules/richaudienceBidAdapter.js

```diff
@@ -263,3 +267,42 @@ function raiGetResolution() {
   }
   return resolution;
 }
+
+function raiGetSyncInclude(config) {
+  try {
+    let raConfig = null;
+    let raiSync = {};
+    if (config.getConfig('userSync').filterSettings != null) {
+      raConfig = config.getConfig('userSync').filterSettings
+      if (raConfig.iframe != null && typeof raConfig.iframe != 'undefined') {
+        raConfig.iframe.bidders == 'richaudience' || raConfig.iframe.bidders == '*' ? raiSync.raiIframe = raConfig.iframe.filter : raiSync.raiIframe = 'exclude';
```

Comment:
Move the assignment outside. A similar change should be made on line 281, too.

```
raiSync.raiIframe = <ternary operator>
```

Reply:
solved